scripts/sphinx_extensions/options_from_kconfig/options_from_kconfig.py

```diff
@@ -84,17 +84,26 @@ def run(self):
                    (prefix.startswith("'") and prefix.endswith("'")):
                     prefix = prefix[1:-1]
                 text += prefix
-            help_ = f'{sym.nodes[0].prompt[0]}'
+            try:
+                prompt_ = f'{sym.nodes[0].prompt[0]}'
```

Comment:
I was under the impression that this was always available so no `try` needed, but this looks better.

Reply:
If config does not have a prompt it is not seen by the user of menuconfig but can be automatically selected. We use it in one or two places.